src/components/datagrid/_datagrid-new.scss

```
@@ -222,6 +228,12 @@
   }
 
   &.medium-rowheight {
+    [class^='btn'] {
```

```suggestion
    [class^='btn'] {
```

Target `row-btn` to be safe

Should we

---

Done